.github/workflows/ios.yml

```diff
@@ -226,7 +226,11 @@ jobs:
             QMAKE_MAC_XCODE_SETTINGS+=qstyle \
             qstyle.name=CODE_SIGN_STYLE \
             qstyle.value=Manual
-
+            
+          # Use old deprecated build system in XCode 13
+          /usr/libexec/PlistBuddy -c "Add :DisableBuildSystemDeprecationDiagnostic bool" ./Input.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
+          /usr/libexec/PlistBuddy -c "Set :DisableBuildSystemDeprecationDiagnostic true" ./Input.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
+              
```

Comment:
Do we need to use the old build system?
Locally I applied this patch and it now generates the new build system https://codereview.qt-project.org/c/qt/qtbase/+/353119

Reply:
if it works without it now, I would keep the old one  for this commit. we should focus on qt6 path rather.

Reply:
Ok, I agree, though it seems that it is neither fixed in qt6 https://bugreports.qt.io/browse/QTCREATORBUG-25368